### DISCUSSION TITLE
Next JS not running

### DISCUSSION THREAD

**Discussion by softcruder** [vercel/next.js#47705] [Help]
### Summary

I'm new to next and i created a new project. I used the tutorial to create a next js with react * typescipt and tried to run with "npm run dev".
The app started
ready - started server on 0.0.0.0:3000, url: http://localhost:3000 
but the server never runs with no errors stated.
i tried to change the port to 1403 and rest network settings but the issue is not fixed
node version v19.7.0
next version 13.2.4

**Comment by icyJoseph** [comment]
Just asking a few troubleshooting questions:

- Did you open your browser after running `npm run dev`? The Next.js dev server doesn't actually compile until a request for a page arrives
- What OS have you got? Sometimes it seems like Windows users end up with Node 32 bits installed on a 64 bits environment and that crashes

**Reply by softcruder** [reply]
Yes, I opened the browser from the terminal.
I use Windows 10, and installed node 64-bits on a 64-bits machine.
